src/components/Form/Form.jsx

```diff
@@ -12,8 +12,11 @@ const isFeedbackEmpty = value => value.length;
 
 const BasicForm = () => {
   const [message, setMessage] = useState('');
-  const [rating, setRating] = useState(null);
   const [formIsValid, setFormIsValid] = useState(false);
+  const [starReview, setStarReview] = useState({
+    stars: [...Array.from({ length: 5 }, (_, i) => i + 1)],
```

Comment:
if `stars` is a variable that never changes, it would be better to store it as constant, not in state object

Reply:
Ok, I will fix it ;) pass it to constants.